wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationMessageSectionControllerTests.swift

```diff
@@ -19,17 +19,21 @@
 import XCTest
 @testable import Wire
 
-final class ConversationMessageSectionControllerTests: XCTestCase {
+final class ConversationMessageSectionControllerTests: ZMSnapshotTestCase {
```

Comment:
Why is this a snapshot test?

Reply:
Another thing, that I committed by accident as I was testing different scenarios. Fixed.